README.md

```diff
@@ -793,6 +781,34 @@ Define a function that returns parameters for filtering API calls. You can use i
                 })
         ]);
 
+## Customizing the API Mapping
+
+All HTTP requests made by ng-admin to your REST API are carried out by [Restangular](https://github.com/mgonto/restangular), which is like `$resource` on steroids.
+
+The REST specification doesn't provide enough detail to cover all requirements of an administration GUI. ng-admin makes some assumptions about how your API is designed. All of these assumptions can be overridden by way of [Restangular's request and response interceptors](https://github.com/mgonto/restangular#addresponseinterceptor).
```

Comment:
I refer to the extensive documentation chapter about API Mapping two lines down, I think it's enough.